augur/data/schema-annotations.json

```diff
@@ -1,34 +1,44 @@
 {
     "type" : "object",
     "$schema": "http://json-schema.org/draft-06/schema#",
-    "title": "JSON object for the `annotations` key, typically produced by `augur translate`",
-    "description": "Coordinates etc of genes / genome",
+    "title": "Schema for the 'annotations' property (node-data JSON) or the 'genome_annotations' property (auspice JSON)",
+    "properties": {
+        "nuc": {
+            "type": "object",
+            "allOf": [{ "$ref": "#/$defs/startend" }],
+            "additionalProperties": true,
+            "$comment": "All other properties are unused by Auspice. Strand is always considered to be positive."
+        }
+    },
+    "required": ["nuc"],
     "patternProperties": {
         "^[a-zA-Z0-9*_-]+$": {
             "type": "object",
+            "allOf": [{ "$ref": "#/$defs/startend" }],
+            "additionalProperties": true,
             "properties": {
-                "seqid":{
-                    "description": "Sequence on which the coordinates below are valid. Could be viral segment, bacterial contig, etc",
-                    "$comment": "Unused by Auspice 2.0",
-                    "type": "string"
-                },
-                "type": {
-                    "description": "Type of the feature. could be mRNA, CDS, or similar",
-                    "$comment": "Unused by Auspice 2.0",
+                "strand": {
+                    "description": "Is the gene on the positive ('+') or negative ('-') strand.",
+                    "$comment": "Auspice assumes positive strand unless strand is '-'",
                     "type": "string"
-                },
+                }
+            }
+        }
+    },
+    "$defs": {
+        "startend": {
+            "type": "object",
+            "required": ["start", "end"],
+            "properties": {
                 "start": {
-                    "description": "Gene start position (one-based, following GFF format)",
-                    "type": "number"
+                    "type": "integer",
+                    "minimum": 1,
+                    "description": "Start position (one-based, following GFF format)"
                 },
                 "end": {
-                    "description": "Gene end position (one-based closed, last position of feature, following GFF format)",
-                    "type": "number"
-                },
-                "strand": {
-                    "description": "Positive or negative strand",
-                    "type": "string",
-                    "enum": ["-","+"]
```

Comment:
Suggestion: Add `enum` back with additional values.

Even though Auspice only cares if the value is `'-'` or not, Augur can also export values `'?'` and `None` as [suggested](https://github.com/nextstrain/augur/pull/966#discussion_r972433558) and implemented in 31f0b2665f962f0e3c3ba2ff59843533365c692c. Defining the possible values here will improve the effectiveness of validation.

I started this in #1279 but that PR can be closed and the change included in here.

Reply:
Thanks for the review -- really appreciated!

We're going to have to think through this. All annotations are interpreted by Auspice as CDSs, so a strand of `"?"` or `"."` ([which we represent](https://github.com/nextstrain/augur/pull/966#discussion_r1296551288) as `None`) doesn't make sense as a CDS - neither for Auspice to display nor for Augur to translate. 

I can allow them in the schema and then have Auspice filter to `["+", "-"]`, which is probably the most technically correct, but I would think that happily translating `"?"` / `"."` features is questionable.

Reply:
Have shifted this conversation to #1279